eureka2-core/src/main/java/com/netflix/eureka2/config/ConfigNameStrings.java

```diff
@@ -0,0 +1,25 @@
+package com.netflix.eureka2.config;
+
+/**
+ * use camel case for better readability
+ *
+ * @author David Liu
+ */
+public final class ConfigNameStrings {
+    private static final String EUREKA_PREFIX = "eurekas.";
+
+    public final class Transport {
```

Comment:
TansportNames? So we can do static import, and have a proper context in the class name.